### DISCUSSION TITLE
Any news/info on the 5-star scheme for best practices using SSSOM?

### DISCUSSION THREAD

**Discussion by WvandenBerg** [mapping-commons/sssom#271] [General]
I've read the following in [A Simple Standard for Sharing Ontological Mappings (SSSOM)](https://academic.oup.com/database/article/doi/10.1093/database/baac035/6591806):

> Developing a metadata standard is usually not enough to improve the quality of data (in our case, mappings) and needs to be accompanied by a set of shared best practices. Analogously to the 5-Star deployment scheme for Linked Data developed by Sir Tim Berners-Lee (https://www.w3.org/2011/gld/wiki/5_Star_Linked_Data), we are developing a 5-Star scheme for rating mappings ([89](javascript:;)). This 5-star scheme directly evolved out of our experiences working with our collaborators to understand how to best document mappings. It includes considerations such as where and how mapping sets should be published, how they should be licensed and which metadata should be provided.

There's also mention of training materials.

I'm very interested in this 5-star scheme you are working on. I've found that the framework of [Five Stars of Linked Data Vocabulary Use](https://www.semantic-web-journal.net/system/files/swj653.pdf) is very helpful in getting non-experts in semantic tech to improve the use of vocabularies for the data that they publish. We are planning to implement some kind of 5-star scheme in our web application where users are encouraged to map their data to common vocabularies.

**Comment by matentzn** [comment]
@WvandenBerg Thank you for reaching out! The team would be happy to hear your feedback on this stuff as well.

Here is the current 5-Star system we propose: https://mapping-commons.github.io/sssom/5star-mappings/ - Note that SSSOM only comes in at 3-Star!

A basic tutorial can be found here: https://mapping-commons.github.io/sssom/tutorial/

But feel free to click around the page to find more resources.

This presentation is probably a good starting point: https://mapping-commons.github.io/sssom/events/ccb2022/ (Other presentations: https://mapping-commons.github.io/sssom/presentations/)

Let me know if we can share anything else!
